CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ release.
 
 ## Unreleased
 
+- BREAKING: Generate process metrics from YAML
+  ([#330](https://github.com/open-telemetry/semantic-conventions/pull/330))
+  - Rename attributes for `process.cpu.time`
```

Comment:
If you want, this can be shorter by doing the same as I did for the system metrics:

```
- Rename attributes for `system.cpu.*` metrics:
    - `state` to `system.cpu.state`
    - `cpu` to `system.cpu.logical_number`
```

Reply:
Resolved in [5ad3a95](https://github.com/open-telemetry/semantic-conventions/pull/330/commits/5ad3a95ab2ddaf681fef56fbe757d1dedda68744) (rebased so it moved to a different place)